docs/introduction/diffusion_intro/diffusion_intro.rst

```diff
@@ -0,0 +1,138 @@
+
+*Note that refining missing frames is necessary for MSD analysis, but not for CVE.* MSD analysis works best when the time between all data points is the same. 
+
+.. image:: figures/refinement.png
+
+Pylake has :ref:`two methods on refinement<localization_refinement>`:
+
+1) **Centroid refinement**
+
+Centroid refinement always refines missing frames. The disadvantage of this method, is that it does not perform well when tracks are close together or with high background. 
+At the moment, this method is the default in the kymotracker.
+
+2) **Gaussian refinement (recommended)**
+
+Gaussidan refinement fits a Gaussian function to improve the position estimate. This method is better at refining tracks that are close together. 
+Using Gaussian refinement to improve the position estimate of tracks is always recommended. Gaussian refinement can also be used to refine missing frames, but this is not the default. When doing MSD analysis, 
+you have to activate the refinement of missing frames, by setting `refine_missing_frames = True` (see section on :ref:`gaussian refinement<kymotracker_gaussian_refinement>`).
+
+
+FAQ on diffusion analysis
+--------------------------
+- **The diffusion coefficients for some of the tracks are negative, what does that mean?**
+    Sometimes a diffusion coefficient of a single track is negative. For example, when your diffusion coefficient is low, you will have some diffusion coefficients above and some below zero. 
+    The negative values do not have a biological meaning by themselves, but the average of all the diffusion coefficients from different tracks should still give a good, positively values estimate of the diffusion 
+    coefficient (provided that you have enough tracks, and that the protein is diffusing freely). The negative data points *should not* be removed from the dataset, otherwise you get a bias in your estimate. 
+
+- **What happens to the diffusion coefficient if a protein bumps into another protein or against other obstacles?**
+    If there are many diffusive proteins binding to the DNA at once and they are not able to pass each other, the proteins will confine the motion, resulting in sub-diffusive or confined diffusive behavior.
+    If the DNA is held between two beads in an optical tweezer experiment, the beads can also confine the motion of a diffusive protein, if the protein diffuses far enough. 
+    If you are interested in free diffusion of a protein, it is best to keep the density of protein binding events low enough, so the proteins don't meet often. 
+    This will help reducing the number binding events, but not when the motion is so wide that the protein ends up on the beads. If you want to exclude the effect of the beads, you can consider cutting the track 
+    to exclude the part where the protein reaches the bead.
+
```

Comment:
Not ignoring it :p

I write that the protein concentration should be 'low enough, such that proteins don't meet often', so this aligns with you saying that crowding only starts to matter once protein bounce into each other a lot.

And since the effect of the beads cannot be accounted for in the analysis in Pylake at the moment, I don't want to go into it too much. I'll just mention that it is possible to account for the beads during analysis. Your explanation about flattening makes a lot of sense. But yes, a reference would be nice...